packages/core/lib/triggers/TRPR0029.test.ts

```diff
@@ -0,0 +1,134 @@
+import TriggerCode from "bichard7-next-data-latest/dist/types/TriggerCode"
+import type { AnnotatedHearingOutcome } from "../../types/AnnotatedHearingOutcome"
+import TRPR0029 from "./TRPR0029"
+
+const generateMockAho = (
+  resultVariableText: string,
+  RecordableOnPNCindicator: string | undefined,
```

Comment:
```suggestion
  recordableOnPNCindicator: string | undefined,
```

Reply:
`RecordableOnPNCindicator` is a boolean type in the schema.

https://github.com/ministryofjustice/bichard7-next-core/blob/annotated-pud-exceptions/packages/core/schemas/unvalidatedHearingOutcome.ts#L335